src/main/java/com/android/volley/toolbox/HurlStack.java

```diff
@@ -223,7 +239,7 @@ private HttpURLConnection openConnection(URL url, Request<?> request) throws IOE
     // NOTE: Any request headers added here (via setRequestProperty or addRequestProperty) should be
     // checked against the existing properties in the connection and not overridden if already set.
     @SuppressWarnings("deprecation")
-    /* package */ static void setConnectionParametersForRequest(
+    /* package */ static void setConnectionParametersForRequest(HurlStack stack,
```

Comment:
Rather than passing HurlStack through static methods, we should probably just make those methods non-static, especially if they're private/package-private helpers.